devicemodel/hw/platform/usb_pmapper.c

```diff
@@ -285,6 +286,7 @@ usb_dev_comp_req(struct libusb_transfer *libusb_xfer)
 				block->blen = 0;
 			}
 		}
+		i++;
```

Comment:
I am a bit lost. The only difference between the old code and new code is in the semantics of "continue" inside the loop.

Can you explain  the detail reason?